### ISSUE TITLE
During provision I am getting this network error.

### ISSUE THREAD

**Issue by sabbir1991** [Varying-Vagrant-Vagrants/VVV#2576]
## What was The Command Used To Provision

```shell
vagrant up --provision
```

## What Kind of VVV Provision Was This

This was a reprovision of an already working VVV

## Steps to Reproduce (for bugs)

1. Changing config.yml for provision 2 site.
2. Then run vagrant up --provision
3. Then after few min getting this network error

Here is the full error log https://gist.github.com/sabbir1991/e64bec9bb2649dbffd4180a0fad3860b

**Comment by Mte90** [comment]
We did an update with the fix few hours ago on the develop branch.

**Comment by tomjn** [comment]
@Mte90 I actually think this is a separate issue that has already been fixed, that of changing `vvv_warning` to `vvv_warn`

@sabbir1991 your error log was incomplete and didn't contain the initial splash screen with the debug data and you didn't include that in your report either. I'm assuming you are not using the latest version? Have you tried the `develop` branch?

**Comment by sabbir1991** [comment]
@tomjn I have updated the gist with full logs https://gist.github.com/sabbir1991/e64bec9bb2649dbffd4180a0fad3860b
I am using vagrant v2.2.19 and vvv v3.8.1

**Comment by sabbir1991** [comment]
@tomjn @Mte90 I have checked using latest develop branch. Everything looks fine. Not getting any issues now. Thanks :)

**Comment by tomjn** [comment]
The PR #2574 has some improvements related to this. The listing of failed checks showed the wrong things, now it uses a simpler loop that only lists what failed.

The actual fix for the reported issue is on the `develop` branch. Once that PR is merged I'm intending to close this and start v3.9 release processes

**Comment by tomjn** [comment]
TLDR use v3.9 when released, or switch to the `develop` branch
